docs/src/content/docs/platforms/desktop.mdx

```diff
@@ -15,6 +17,85 @@ are supported by their vendors at the time of a Slint version release.
 | Windows 10       | x86-64       |
 | Windows 11       | x86-64       |
 
+### Handle the console window
+
+When you run an application built by slint, a console window would show, by default ( see [/SUBSYSTEM (Specify Subsystem)](https://learn.microsoft.com/en-us/cpp/build/reference/subsystem-specify-subsystem?view=msvc-170)).
+
+You could disable the console by specify window subsystem.
+
+<Tabs syncKey="dev-language">
+<TabItem label="Rust">
+
+Add the code to the top of .rs file which contains `fn main()`.
+
+```rust
+#![windows_subsystem = "windows"]
+```
+
+Or if you want to keep console output in debug mode:
+
+```rust
+#![cfg_attr(not(debug_assertions), windows_subsystem = "windows")]
+```
+</TabItem>
+<TabItem label="C++">
+
+Add link flag `/SUBSYSTEM:WINDOWS`. If you are using Visual Studio, follow the steps:
+
+1. Open the project's Property Pages dialog box. For details, see Set C++ compiler and build properties in Visual Studio.
+
+2. Select the Configuration Properties > Linker > System property page.
+
+3. Modify the SubSystem property, select "Window(/SUBSYSTEM:WINDOWS)".
+
+</TabItem>
+<TabItem label="Node.js">
+
+TODO: Add node.js
+</TabItem>
+</Tabs>
+
+Set subsystem to "windows" would lost stdout if you run the application from command line. If you want to deal with CLI, consider use `FreeConsole()` instead of flag subsystem.
+
+Here is a Rust example. The parent process should be "explorer.exe" if you run the application from desktop, or folder window, so we could free the console as we dont want to see it.
```

Comment:
Again this is hard to parse. "The following Rust example.. <does?>". I'm the UI person here so my knowledge of what is being solved here is limited! Is it something like "The following Rust example detects where the app is being run from. If run from the command line `FreeConsole()` is used to let it output debug info while not outputting anything when run as a traditional app."

Reply:
Partially right. If run from the command line, we don't use `FreeConsole()` because we need it. Only use `FreeConsole()` when run as a traditional app, so the console window will be closed.

Reply:
"Here is a Rust example. It checks if the parent process is "explorer.exe" to ensure the console window is closed when opening the app via the normal windows GUI". Am I understanding now?

Reply:
So you want to make these tweaks? Then we get this approved and merged in.

Reply:
Sure, I used your improvement and submitted it